main_location.py

```diff
@@ -0,0 +1,106 @@
+import csv
+from dataclasses import dataclass
+
+
+@dataclass
+class Local_Weather:
```

Comment:
Small thing, but when you're naming classes you should use camel case, i.e., `Local_Weather` should be `LocalWeather`